infra/scripts/test_configure.sh

```diff
@@ -64,21 +80,68 @@ while true; do
     shift
 done
 
+# Check torch version
 if [ "$_TORCH_VER" != "2.5" -a "$_TORCH_VER" != "2.6" -a "$_TORCH_VER" != "nightly" ]; then
     echo "Invalid '$_TORCH_VER'"
     echo "(Use --help to see available options)"
     exit 1
 fi
+# Check if torch version is nightly
+if [[ "$_TORCH_VER" == "nightly" ]]; then
+  _NIGHTLY=true
+fi
+# Check for conflicting options
+if [[ -n "$_USER_SPECIFIED_CUDA" && "$_CPU_ONLY" = true ]]; then
+  echo "[ERROR] Cannot use --cpu_only and --cuda_ver together"
+  exit 1
+fi
+
+# set index-url
+get_index_url_for_cuda_version() {
+  local version="$1"
+  local is_nightly="$2"
+
+  case "$version" in
+    11.8) echo "https://download.pytorch.org/whl${is_nightly:+/nightly}/cu118" ;;
+    12.1) echo "https://download.pytorch.org/whl${is_nightly:+/nightly}/cu121" ;;
+    *) echo "" ;;
```

Comment:
Don't we support other cuda versions?

I have DETECTED_CUDA==12.6
And `get_index_url_for_cuda_version` function returns nothing, it gives me void INDEX_URL.

Reply:
Ah, I missed that version. I'll just use [11.8, 12.6, 12.8] according to [the torch guide](https://pytorch.org/get-started/locally/).